### PR TITLE
[INLONG-2075] SDK Source of DataProxy support new Message format.

### LINKED ISSUE #2075: [Feature] SDK Source of DataProxy support new Message format.
### Description

new Message format
<html>
<body>
<h2 id="messageFormat" style="margin: 30px 0px 0px; padding: 0px; color: rgb(23, 43, 77); font-weight: normal; text-transform: none; font-size: 20px; line-height: 1.5; letter-spacing: -0.008em; border-bottom-color: rgb(255, 255, 255); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen, Ubuntu, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; orphans: 2; text-indent: 0px; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; text-align: left;"><span class="nh-number" style="margin: 0px;"></span>Message format</h2><div class="table-wrap" style="margin: 10px 0px 0px; padding: 0px; overflow-x: auto; color: rgb(23, 43, 77); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen, Ubuntu, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

MessageContext | Type | Remark
-- | -- | --
headers | Map<String,String> |  data parameters that user sends.
body | byte[] | Raw data that user sends.

</div>
</body>
</html>

<html>
<body>
<h2 class="auto-cursor-target" id="id-headers" style="margin: 30px 0px 0px; padding: 0px; color: rgb(23, 43, 77); font-weight: normal; text-transform: none; font-size: 20px; line-height: 1.5; letter-spacing: -0.008em; border-bottom-color: rgb(255, 255, 255); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen, Ubuntu, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; orphans: 2; text-align: start; text-indent: 0px; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">headers format</h2><div class="table-wrap" style="margin: 10px 0px 0px; padding: 0px; overflow-x: auto; color: rgb(23, 43, 77); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen, Ubuntu, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Header name | Remark
-- | --
inlongGroupId | inlongGroupId
inlongStreamId | inlongStreamId
topic |  Topic name of Cache Cluster(Kafka, Pulsar, Tube)
msgTime | time in milliseconds when user sends message data
sourceIp | node IP that user sends message date

</div>
</body>
</html>

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by luchunliang**
### Title Name: [INLONG-2075][Inlong-DataProxy] SDK Source of DataProxy supports new Message format.
Fixes #2075 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by gosonzhang — APPROVED** [review]
+1

**Review by dockerzhang — APPROVED** [review]

**Comment by baomingyu** [comment]
+1

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source/ServerMessageHandler.java [hunk 1]
```diff
@@ -408,49 +417,72 @@ private void formatMessagesAndSend(Map<String, String> commonAttrMap,
...
-                } catch (Exception e1) {
-                    long uniqVal = Long.parseLong(commonAttrMap.get(AttributeConstants.UNIQ_ID));
-                    throw new MessageIDException(uniqVal,
-                        ErrorCode.DT_ERROR,
-                        new Throwable("attribute dt=" + headers.get(AttributeConstants.DATA_TIME
-                            + " has error, detail is: topic=" + topicEntry.getKey() + "&streamId="
-                            + streamIdEntry.getKey() + "&NodeIP=" + strRemoteIP), e1));
-                }
+                // process proxy message list
+                this.processProxyMessageList(headers, streamIdEntry.getValue());
```
Comment (baomingyu) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source/ServerMessageHandler.java:427]: the message of tdmsg format is deleted?  the tdmsg format needs to be retained. And new message format need add tow new file for messagehandler and sink.

**Review by baomingyu — CHANGES_REQUESTED** [review]

**Comment by baomingyu** [comment]
+1

**Review by baomingyu — APPROVED** [review]

**Review by baomingyu — APPROVED** [review]

**Pull request merged**